### ISSUE TITLE
Persian text displays wrong in list view

### ISSUE THREAD

**Issue by v187077924** [URUWorks/TeroSubtitler#244]
Hi, In the list view, as for translation or source sub when you import or create a Persian subtitle it displays it wrongly, it is fine in both video and text box but not in the list view.  Please fix it, Thx.
Also I couldn't find any setting for the font size, it's rather small, Can we change the font? if Yes, Please tell me wehere.

here is the picture for better understanding:

![image](https://github.com/URUWorks/TeroSubtitler/assets/35186406/4e1e2b94-deb3-4046-ae84-1cd94f53de19)

**Comment by chenlung** [comment]
Pertains to the handling of RTL text?

**Comment by v187077924** [comment]
> Pertains to the handling of RTL text?

Yes, Probably

**Comment by URUWorks** [comment]
Correct, it has to be fixed, a quick solution: open Settings and uncheck "Interpret tags in list".
About font size, Settings/Appearance, "List font size" and/or "Text box font size".

**Comment by v187077924** [comment]
Thank you So much, Yeah it's fixed now
